lenskit-gradle/src/main/groovy/org/lenskit/gradle/LenskitExtension.groovy

```diff
@@ -72,4 +72,14 @@ public class LenskitExtension {
      * and the log file.
      */
     def String logFileLevel
+
+    /*
+* The list of jvm argument prroperties
+* */
+    def List<String> jvmArgs
+
+    def addJVMArgs(String val){
```

Comment:
A few changes we need here:

- This should take an array of strings (`String... args`).
- It should just be called `jvmArgs`.
- It needs to add the arguments to the `jvmArgs` list, not just return them.